src/components/PageTitleBar/PageTitleBar.story.jsx

```diff
@@ -24,6 +24,13 @@ export const commonPageTitleBarProps = {
   ),
 };
 
+const NodeTooltip = () => (
+  <div>
+    Descriptive text about this page and what the user can or should do on it{' '}
+    <button type="button">Even Buttons</button>
```

Comment:
Could we use the Carbon button to show a more visually valid example?

Current storybook deploy preview:
![image](https://user-images.githubusercontent.com/3360588/78943739-06676d00-7a82-11ea-8e2a-8db566f67e35.png)
Design guidance from root issue:
![image](
https://camo.githubusercontent.com/9fb870f7ffc091b3050654474b9f9e56af3c73cd/68747470733a2f2f6d656469612e6769746875622e69626d2e636f6d2f757365722f3833372f66696c65732f31383863386330302d363166342d313165612d396566622d653837376138353261333064)